zebra/zebra_ns.c

```diff
@@ -179,9 +181,11 @@ unsigned long zebra_ns_score_proto(uint8_t proto, unsigned short instance)
 
 	zns = zebra_ns_lookup(NS_DEFAULT);
 
-	RB_FOREACH (znst, zebra_ns_table_head, &zns->ns_tables)
+	RB_FOREACH (znst, zebra_ns_table_head, &zns->ns_tables) {
+		if (znst->ns_id != NS_DEFAULT)
+			continue;
```

Comment:
question: why is this exception necessary here and in the other `RB_FOREACH` loops?

Reply:
this is to maintain functional perimeter